templates/docs/examples/patterns/navigation/deprecated-logo.html

```diff
@@ -148,10 +148,12 @@
   </div>
 </header>
 
-<script>
-  {% include 'docs/examples/patterns/navigation/_script-search.js' %}
-  {% include 'docs/examples/patterns/navigation/_script.js' %}
+{% if not is_combined %}
```

Comment:
A possible approach could be to have `{% block script %}` in `examples.html`, that would output only in non-combined examples. And then in individual templates we put scripts into that.

But that will mean that none combined examples would have scripts, even if they may work fine.

Reply:
@bartaz We can set things up with `block script` and another flag like `ignore_script` so that we by-default embed the scripts block. Then for special cases like navigation we can set `ignore_script` to true in its example so that the examples template does not embed the script. 

If we want the possibility of the combined examples working with scripts, we have to add something to some of the examples. Is that changed approach minimal enough to make the barrier for entry acceptable?

Reply:
> If we want the possibility of the combined examples working with scripts, we have to add something to some of the examples. Is that changed approach minimal enough to make the barrier for entry acceptable?

I don't know. But, as for now, we only need combined examples for Percy anyway. So it should be OK to simply remove scripts from the examples.
So I'd suggest adding `scripts` block (similar to what we have in `_root.html`) to example template and not render contents of it in combined example. So that for navigation we can put the scripts in the block.
But don't refactor any other examples yet, not to blow up the scope and cause conflicts.

If we get to the point when we want to add scripts to combined examples we can figure out solution then. A simple workaround would be to put scripts in body block, as they currently are.

Reply:
@bartaz Revised to use script block as suggested